src/shared/timeseries/runCrawler.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const childProcess = require('child_process');
+
+/**
+ * Running crawler for each day in its own process.
+ *
+ * This will result in a slower execution time due to the child_process overheads,
+ * but it will ensure that timeseries never runs out of memory.
+ */
+export default options =>
+  new Promise((resolve, reject) => {
+    const child = childProcess.fork(
+      path.join('src', 'shared', 'timeseries', 'worker.js'),
+      [
+        '-d',
+        options.date,
```

Comment:
Can you also pass `-e` and `-o` and `-q` and really all of them?

Reply:
Done!